Formula/guichan.rb

```diff
@@ -25,7 +25,7 @@ class Guichan < Formula
   end
 
   resource "fixedfont.bmp" do
-    url "https://guichan.sourceforge.io/oldsite/images/fixedfont.bmp"
+    url "http://guichan.sourceforge.io/oldsite/images/fixedfont.bmp"
     sha256 "fc6144c8fefa27c207560820450abb41378c705a0655f536ce33e44a5332c5cc"
   end
```

Comment:
Do we need this change in your PR?

Reply:
I see a redirect to http in a browser and curl SSL error on the Linux runner.

Reply:
We can do it in a separate PR so we don't have to republish this formula's bottles.

Reply:
Here you go: #108581